rst2pdf/findfonts.py

```diff
@@ -193,6 +194,8 @@ def get_family(query):
 
     def get_fname(query):
         data = os.popen("fc-match -v \"%s\""%query, "r").read()
```

Comment:
maybe change this to use subprocess.check_output? Just so it looks less 2010 ish :-)